cpp/foxglove/tests/test_mcap.cpp

```diff
@@ -118,8 +118,10 @@ TEST_CASE("different contexts") {
   auto context2 = foxglove::Context::create();
 
   // Create writer on context1
-  foxglove::McapWriterOptions options{context1};
```

Comment:
Initializing `McapWriterOptions` and passing only the context in the initializer caused warnings because not every struct member is being initialized. I experimented with using designated initializers, but that required C++20 support and we're building with C++17.